New_PlatformGame/Assets/1. Script/MonsterCtrl.cs

```diff
@@ -33,7 +33,8 @@ void OnTriggerEnter2D(Collider2D col)
         if (col.CompareTag("Player"))
         {
             Debug.Log($"player : {col.name}");
-            Destroy(col.gameObject);
+            /*Destroy(col.gameObject);*/
```

Comment:
```suggestion
            Destroy(col.gameObject);
```
왜 같은 코드 두 줄이 한 줄은 주석 처리가 되어있죠?